### DISCUSSION TITLE
Error when creating new scenario

### DISCUSSION THREAD

**Discussion by pagarsimaz** [frePPLe/frepple#668] [Q&A]
Good morning, 
we have installed Frepple on premise and when I try to create a new scenario I get the following error: "connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL: Peer authentication failed for user "frepple"". 
We probably made some mistakes during the configuration phase. Can anyone help us? Thank you

**Comment by jdetaeye** [comment]
- If you are running on ubuntu 24:
   Make sure that the /etc/frepple/djangosettings are correctly filled out for all scenarios.
   USER, NAME, PASSWORD, HOST and PORT need to be set correctly here: https://github.com/frePPLe/frepple/blob/934ea78555df83ee68cb29308e968249d7e95c5c/djangosettings.py#L50
     
- If you are running a container:
   Pass the variables POSTGRES_DBNAME, POSTGRES_HOST, POSTGRES_USER and POSTGRES_PASSWORD to your container.

**Comment by pagarsimaz** [comment]
We are in the first case (ubuntu 24). 
We followed your suggestion and solved the problem.
Thank you very much!
